taichi/codegen/codegen_llvm.cpp

```diff
@@ -868,7 +875,7 @@ void CodeGenLLVM::visit(ArgLoadStmt *stmt) {
 }
 
 void CodeGenLLVM::visit(KernelReturnStmt *stmt) {
-  if (stmt->is_ptr) {
+  if (stmt->ret_type.is_pointer()) {
```

Comment:
This is the only substantial change in this file. Note that this part of the code is never used yet anyway.